cypress/e2e/spec.cy.js

```diff
@@ -132,4 +132,48 @@ describe('Login Tests', () => {
 
     cy.contains('Password is too short.').should('be.visible')
   })
+
+  it('should display error alert when failed login', () => {
+    cy.visit(Cypress.env('BASE_URL'))
+
+    cy.contains('UnReveal').should('be.visible')
+    cy.contains('Login').should('be.visible')
+    cy.contains('Register').should('be.visible')
+
+    cy.contains('Login').click()
+
+    cy.get('[placeholder="Email"]').type('email@email.com')
+    cy.get('[placeholder="Password"]').type('password')
+
+    cy.intercept(`*login`, {
+      statusCode: 401,
+    })
+
+    cy.contains("Let's go!").click()
+
+    cy.on('windows:alert', (str) => {
+      expect(str).to.equal('Email already taken or invalid inputs')
+    })
+  })
+
+  it('should display error alert when failed login_404', () => {
+    cy.visit(Cypress.env('BASE_URL'))
+
+    cy.contains('UnReveal').should('be.visible')
+    cy.contains('Login').should('be.visible')
+    cy.contains('Register').should('be.visible')
+
+    cy.contains('Login').click()
+
+    cy.get('[placeholder="Email"]').type('email@email.com')
+    cy.get('[placeholder="Password"]').type('password')
+
+    cy.contains("Let's go!").click()
+
+    cy.request({
```

Comment:
The test should not make requests, the app is the one who does it. The test should intercept requests

Reply:
@SaraGrimaldi